src/views/Error/index.scss

```
@@ -0,0 +1,26 @@
+@use "/src/stylesheets/responsiveness";
+
+.error-view {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-evenly;
+    background: linear-gradient(#8296D9,#BFACF0);
+    @include responsiveness.font(10vw, 72px, 144px);
+    .error-view-message {
```

```suggestion
    &__message {
```

Consider using the [BEM](http://getbem.com/naming/) naming convention. e.g.: `&__message` here in SCSS would correspond to the `.error-view__message` class name.

I think this will improve maintainability and avoid naming collisions (not that this will happen in this view, but more likely in the Home view for common names).